doc/source/matching_strategies.rst

```diff
@@ -40,6 +40,105 @@ Grid-fees are calculated for each order using the following grid fee matrix:
 | Origin Cluster: 1  | 1                        | 0                       |
 +--------------------+--------------------------+-------------------------+
 
+.. _grid_fee_matrix:
+
+Methods to Enter Grid Fee Matrix
+--------------------------------
+
+There are two primary methods to input a grid fee matrix into the Simply Simulation:
+
+1. Providing a Complete Grid Fee Matrix
+2. Building a Grid Fee Matrix from Configuration Files
+
+Providing an Already Complete Grid Fee Matrix
+---------------------------------------------
+
+Format
+~~~~~~
+
+- The grid fee matrix should be provided in a structured format such as JSON or CSV.
+
+Placement
+~~~~~~~~~
+
+- Place the complete grid fee matrix in the designated input location within the simulation environment, as specified in the simulation documentation or configuration settings.
+
+Requirements
+~~~~~~~~~~~~
+
+Names/Identifiers
+"""""""""""""""""
+
+- Each row and column must be labeled with unique identifiers representing different clusters or nodes.
+- Example Identifiers:
+  - **Network Lines:** line_1, line_2, line_n
+  - **Nodes or Clusters:** node_A, node_B, node_C
+
+Units
+"""""
+
+- Specify consistent units for both row and column headers to avoid confusion (e.g., USD, EUR).
+
+Weight Factor
+"""""""""""""
+
+- The weight factor describes the relationship between grid fee and cumulative power network edge weights.
+
```

Comment:
too many section, does not look very compact, maybe try
- `**Format**: The grid fee matrix should be provided in a structured format such as JSON or CSV.` etc.
- or as a two column table